### DISCUSSION TITLE
Server list not fully populating

### DISCUSSION THREAD

**Discussion by eajohnson70** [jamulussoftware/jamulus#2166] [Technical Support & Help]
Beginning yesterday afternoon, the Jazz genre server list is not fully populating (including my own server). I've tested this on my daily machine and secondary machine (both Macs on the same network) as well as a new install on a Windows PC on a different network. Is anyone else seeing this behavior?

**Comment by pljones** [comment]
Is it visible correctly on http://explorer.jamulus.io ?  If so, it's a client-side issue, most likely a client ISP issue.

**Reply by eajohnson70** [reply]
It's not correct on there either

**Reply by eajohnson70** [reply]
I've tested it on different ISPs as well

**Reply by pljones** [reply]
I can see that the lists look shorter than they were, indeed.

- What `--directoryserver`/`-e` value are you specifying?
- What server version?

**Reply by eajohnson70** [reply]
On the client side, I'm just selecting Genre Jazz in the GUI.

**Reply by pljones** [reply]
No, that's not what I asked.  I asked about the server.  You said you had a server and that it wasn't showing.  So tell me how it's set up.

**Reply by eajohnson70** [reply]
CENTRALSERVER=jamulusjazz.fischvolk.de:22324

**Reply by softins** [reply]
It looks like the name `jamulusjazz.fisckvolk.de` is currently not resolving. Try using `jazz.jamulus.io` instead.

**Reply by pljones** [reply]
Yes, that's a _very_ old name for the server.  (Well, in terms of how often they've been changed -- which, for domain names, has been pretty often.)

**Reply by eajohnson70** [reply]
Changing the domain name in my server config solved the issue for my server. There must be many more servers out there using the deprecated fisckvolk.de fqdn. I'm guessing that allowing the repo to handle updates is the reason for these addresses sticking in the config?

**Reply by pljones** [reply]
> I'm guessing that allowing the repo to handle updates is the reason for these addresses sticking in the config?

I don't get your meaning.

**Reply by eajohnson70** [reply]
The config doesn't get updated with the new fqdns when the server version gets updated because it's a config file with individual settings. I'm assuming that a new server install has the correct fqdn?

**Reply by softins** [reply]
I think the refrence is to installing updates directly from the deb files. They will not update configs that may contain customisations, so it's quite possible that older server names continue to get used until they stop working.

**Reply by eajohnson70** [reply]
Correct. Is this advertised somewhere and many of us missed the announcement?

**Reply by softins** [reply]
See my reply in #2167. For the future, it is probably best to avoid opening both an Issue and a Discussion about the same topic, to avoid fragmenting of the conversation.

**Reply by eajohnson70** [reply]
My apologies, I thought I was moving from a discussion to an issue but see that is not what happened.

**Reply by softins** [reply]
This should now be resolved. It turned out to be an oversight in DNS reorganisation within the last couple of days, which has now been corrected.
